linux_pipeline/Jenkinsfile_azure_latest_image

```diff
@@ -117,6 +118,52 @@ azureImages.each {
         parallel ARMImage
     }
 
+    stage("SGX-${it}") {
+        def currentImage = "${it}".trim()
+        ARMImage = [:]
+
+        sgxRegion.each {
+            def currentRegion = "${it}"
+            if (currentImage.contains("gen2") && (currentImage.contains("18_04-lts-gen2") || currentImage.contains("16_04-lts-gen2"))) {
```

Comment:
does this mean we ignore the possible distro ""Canonical UbuntuServer 19_10-daily-gen2 latest""?